packs/azure/actions/lib/formatters.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    'to_node_dict'
+]
+
+
+def to_node_dict(node):
```

Comment:
That's a possibility, but it doesn't (and probably shouldn't) be available to all the packs. That's why I was thinking about going with a package which is explicitly added to `requirements.txt` inside the packs which need it.